### PR TITLE
added github button doc

### PR DESCRIPTION
**Pull request by bblodfon**

### CODE REVIEW

File: inst/examples/03-formats.Rmd [hunk 1]
```diff
@@ -103,6 +103,7 @@ bookdown::gitbook:
       size: 2
     sharing:
       facebook: yes
+      github: yes
```
Comment (yihui) [inline inst/examples/03-formats.Rmd]: The default is `no`:

```suggestion
      github: no
```

Comment (bblodfon) [reply inst/examples/03-formats.Rmd]: Yes, true! I just took it from my use-case where I wanted just this button and nothing else

Comment (yihui) [reply inst/examples/03-formats.Rmd]: Would you change it to `no`?

Comment (bblodfon) [reply inst/examples/03-formats.Rmd]: Sorry. new to pull requests. Is everything ok now?

**Pull request merged**